### PR TITLE
Add syntax highlighting to readme

### PR DESCRIPTION
**Pull request by zbarbuto**
Makes it slightly prettier and slightly easier to read 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,50 +11,57 @@ Adds a server side check for a mandatory app update and server-side configuratio
...
+  }
+}
+```
 
 Running ManUp will download this file and compare it to the installed app's version to determine if there is an update available (`latest`), or if there is a mandatory update required (`minimum`).
 
 #### Swift
 
+```swift
 	@import ManUp
```
Comment (zbarbuto) [inline README.md]: n.b. this issue is fixed in my other PR https://github.com/NextFaze/ManUp/pull/22

**Review by ricsantos — APPROVED** [review]
🎃

**Comment by swoolcock** [comment]
Resolve merge conflicts and looks good.

**Comment by zbarbuto** [comment]
Have also removed extra level of indentation no longer required.

**Pull request merged**